pom.xml

```diff
@@ -25,19 +25,19 @@
     <version>4.0.0-SNAPSHOT</version>
     <name>Jakarta Enterprise Beans Parent</name>
     <description>Jakarta Enterprise Beans Parent</description>
-    <url>https://github.com/eclipse-ee4j/ejb-api</url>
+    <url>https://github.com/jakartaee/enterprise-beans</url>
     <organization>
         <name>Eclipse Foundation</name>
         <url>https://projects.eclipse.org/projects/ee4j.ejb</url>
     </organization>
     <issueManagement>
         <system>github</system>
-        <url>https://github.com/eclipse-ee4j/ejb-api/issues</url>
+        <url>https://github.com/jakartaee/enterprise-beans/issues</url>
     </issueManagement>
     <scm>
-        <connection>scm:git:ssh://git@github.com/eclipse-ee4j/ejb-api.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/ejb-api.git</developerConnection>
-        <url>https://github.com/eclipse-ee4j/ejb-api/tree/${project.scm.tag}</url>
+        <connection>scm:git:ssh://git@github.com/jakartaee/enterprise-beans.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jakartaee/enterprise-beans.git</developerConnection>
+        <url>https://github.com/jakartaee/enterprise-beans/tree/${project.scm.tag}</url>
```

Comment:
```suggestion
        <url>https://github.com/jakartaee/enterprise-beans</url>
```
I had added the ${project.scm.tag} hoping it would get replaced with the tagged version. for e.g., https://github.com/jakartaee/enterprise-beans/tree/4.0.0-RELEASE

This does not seem to work as per expectation, hence can be removed.

Reply:
> This does not seem to work as per expectation, hence can be removed.

Depends on expectation. It works per mine fine:

![ejb-url](https://user-images.githubusercontent.com/11896137/154268450-87ddc4f2-c138-40fa-8089-62e9ea779c52.png)


> I had added the ${project.scm.tag} hoping it would get replaced with the tagged version. 

Why would you add it? It's already there, with the `HEAD` value now.

It seems to be convention in maven world to refer either to `HEAD` or to specific version if documentation is released.
`scm.tag` element is updated by `maven-release-plugin` for example.

Reply:
Sonatype uses property even in their example: https://central.sonatype.org/publish/requirements/#scm-information [Git on Apache Git Server from Apache Maven]

You could raise a ticket https://issues.sonatype.org/ for them to filter that in search results.

Reply:
I think having the project home page should suffice for now, and so I removed tag element from the url.